### DISCUSSION TITLE
umi3.x打包完devScripts.js modules[moduleId].call 最后一个参数后面多了逗号',' ，导致在IE11下面报错

### DISCUSSION THREAD

**Discussion by ZhaoXueJing** [umijs/umi#6245] [General]
http://localhost:8001/@@/devScripts.js
报错位置：25行第5列

**Comment by sorrycc** [comment]
给最小复现，我本地试最简项目跑 ie 11 没有报错。

**Comment by jyxu1016** [comment]
按照umi快速上手初始化@umijs/umi-app， start之后在ie11 下报错，页面空白
![image](https://user-images.githubusercontent.com/84429503/126598716-aea50f48-1103-46e8-bb04-be4439733192.png)
![image](https://user-images.githubusercontent.com/84429503/126598920-099aaeff-3fd2-4781-9c90-60489d113230.png)

@sorrycc
